### PR TITLE
Scoring fixes 

### PR DESCRIPTION
**Pull request by priyakasimbeg**
Scoring fixes, preliminary to issues/555, issues/556 and issues/557.

### CODE REVIEW

File: scoring/scoring.py [hunk 1]
```diff
@@ -34,6 +34,7 @@
 import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
+from tabulate import tabulate
```
Comment (runame) [inline scoring/scoring.py:37]: Where is this used?

**Review by runame — DISMISSED** [review]
Besides my question, LGTM.

File: scoring/scoring.py (hunk 1 above)
Comment (priyakasimbeg) [reply scoring/scoring.py:37]: Ah good catch, thanks, I used that for debugging I think. Removing.

File: setup.cfg [hunk 2]
```diff
@@ -37,6 +37,7 @@ install_requires =
   absl-py==1.4.0
   numpy>=1.23
   pandas>=2.0.1
+  tabulate==0.9.0
```
Comment (runame) [inline setup.cfg:40]: Is it used anywhere else in the repo? Couldn't find anything with a quick search.

**Review by runame — COMMENTED** [review]
See my comment.

**Pull request merged**